src/utils/ParseRouteUtils.js

```diff
@@ -150,8 +150,14 @@ async function condenseRoute(
       return null;
     }
 
-    // Discard routes where not possible to walk to bus given departure buffer
-    if (departureDelayBuffer && direction.type === 'depart' && startTime < departureTimeNowMs) {
+    /**
+     * Discard routes where not possible to walk to bus given departure buffer
+     * We subtract (ONE_MIN_IN_MILLISECONDS - 1) from departureTimeNow to avoid the
+     * case where we don't show a bus leaving at the same minute as departureTime.
+     */
+    if (departureDelayBuffer
+      && direction.type === 'depart'
+      && startTime < departureTimeNowMs - (ONE_MIN_IN_MILLISECONDS - 1)) {
```

Comment:
that's kind of a long name.. why not just ONE_MIN_IN_MS

Reply:
will change, i did not write this file 😠